client/src/HomePage.js

```diff
@@ -0,0 +1,173 @@
+         <h1 class="intro mr-4">The simplest way to find and organize notes.</h1>
+         <p class="intro-two ml-5 pl-5">AllNotes helps you capture idea and find them fast.</p>
+         <button className="btn btn-outline-info mt-3 py-2 px-5 rounded"><strong>Sign up for free</strong></button>
+         <p class="intro-three mt-2 pt-1">Already have an account ? <span class="text-pink">Log in.</span></p>
+      </div>
+   </div>  
+</div>
+
+<h1 class="intro text-pink mt-5">Comprehensive underneath,</h1>
+<h1 class="intro text-pink">simple on the surface!</h1>
+
+<div className="container mb-5 mt-5">
+<div className="jumbotron">
+  <div className="row">
+    <div className="col-lg-4">
+      <img src="../1.png"/>
+      <h5 className="text-dark pb-1">Use it anywhere</h5>
+      <p className="text-muted mt-1">Notes stay updated across all your devices, automatically and in real time. There's no "sync" button. It just works.</p>
+    </div>
+    <div className="row">
+      <div className="col-lg-5">
+      </div>
+    </div>
+    <div className="col-lg-4">
+      <img src="../2.png"/>
+      <h5 className="text-dark pb-1">Stay organized</h5>
+      <p className="text-muted mt-1">Add tags to find notes quickly with instant searching.</p>
+    </div>
+    <div className="col-lg-4">
+      <img src="../3.png"/>
+      <h5 className="text-dark pb-1">Work together</h5>
+      <p className="text-muted mt-1">Share a to-do list, post some instructions, or publish your notes online.</p>
+    </div>
+  </div>
+  <div className="row mt-5">
+    <div className="col-lg-4">
+      <img src="../4.png"/>
+      <h5 className="text-dark pb-1">Go back in time</h5>
+      <p className="text-muted mt-1">Notes are backed up with every change, so you can see what you noted last week or last month.</p>
+    </div>
+    <div className="row">
+      <div className="col-lg-5">
+      </div>
+    </div>
+    <div className="col-lg-4">
+      <img src="../5.png"/>
+      <h5 className="text-dark pb-1">Markdown support</h5>
+      <p className="text-muted mt-1">Write, preview, and publish your notes in Markdown format.</p>
+    </div>
+    <div className="col-lg-4">
+      <img src="../6.png"/>
+      <h5 className="text-dark pb-1">It’s free!</h5>
+      <p className="text-muted mt-1">Apps, backups, syncing, sharing – it’s all completely free.</p>
+    </div>
+  </div>
+</div>
+</div>
+
+<div className="container-fluid mt-5 ml-3 mb-4">
+   <div className="row">
+      <div className="col-lg-5 pl-5 mt-5">
+         <h1 class="intro-s1 ml-5">Available on all your devices</h1>
+         <p class="intro-s2 ml-2 pl-5 mt-1 pt-2">Sign up today on All Notes for any device and stay in sync - all the time, 
+everywhere.</p>
+         <button className="btn btn-outline-info mt-3 py-2 px-5 rounded"><strong>Sign up for free</strong></button>
+      </div>
+      <div className="col-lg-7 pr-5">
+         <img src="../img2.png" height="400" width="600" />
```

Comment:
Replace this with the designs made by Pooja which are available on the `design` directory. We cannot use someone else's design here :smile: